odb_extractor_atoms.py

```diff
@@ -392,7 +825,7 @@ def readseq(file, path):
     # Read and parse csv file: first line is a comment, second has column headings
     nline = 0
     for line in f:
```

Comment:
This should probably be changed to:
```
for nline, line in enumerate(f):
```
and then we can eliminate the initialization and management of `nline`.